README.TXT

```diff
@@ -1,237 +1,246 @@
-* Fix: The admin menu icons didn't show when used with some SSL certificates.
-* Fix: An image wasn't always getting removed from listings in the back-end, if the image was removed and then the listing was saved.
-* Fix: Show the "Submit listing instructions" message on the first page only.
-* Fix: The NavXT Breadcrumbs for tag pages weren't showing up correctly.
-* Update a few deprecated jQuery warning messages.
-* Include total collected payment amounts in user tracking. This is 100% anonomous and does not include any payment details.
-
-= 6.2.7 =
-* New: Add more currencies including a custom currency option. Simplify the settings for currencies.
-* New: Allow child pages to be created under the main page.
-* Hide the main slug setting when it's not needed.
-* New: Add wpbdp_use_single hook to tell themes to use the category template page instead of the page template.
-* Better integration with Breadcrumbs NavXT: keep the home page link, prevent the BD page from showing twice with some settings, and use a link on the BD page.
-* Fix: Correctly process shortcodes and apply filters in content when the settings are on in the field.
-* Fix: The Elementor page content wasn't able to override the BD content.
-* Fix: Load Divi styling on category pages too.
-
-= 6.2.6 =
-* Fix: Fatal error on category pages when CPT integration was disabled.
-* Fix: Some themes (like Divi) showed repeat content on category pages.
-
-= 6.2.5 =
-* If thumbnails are turned off, and images are showing in a gallery with the listing, then show the thumbnail image there too.
-* Set payment date and time to read only if there's a subscription.
-* On Auth.net checkout page, place exp month and year in same line.
-* Optimize loading time for sites with a lot of users, but no user matching the admin email address for the site.
-* Add new hooks to change image sizes for images from fields: wpbdp_img_width and wpbdp_img_size
-* Fix: Save the payment date if it's changed manually.
-* Fix: Some themes were showing no content for a single listing.
-* Fix: Show the correct category name in Block themes too.
-* Fix: Save the thumbnail and save reordering images from the admin area.
-* Fix: Correctly show validation messages for a required checkbox.
-* Fix: Allow the same value to by used in multiple checkbox fields.
-* Fix: Some sites were getting an error during the database upgrade process.
-* Fix: Show the current variable plan price for logged out users.
-
-= 6.2.4 =
-* Fix: Redirect to the login page from the new listing form when logging in is required.
-* Fix: The listings weren't showing alphabetically in new installations, if the sorting settings hadn't been updated.
-
-<a href="https://businessdirectoryplugin.com/plugins/business-directory/?utm_source=wprepo&utm_medium=link&utm_campaign=liteversion">See changelog for all versions</a>
+=== Business Directory Plugin - Easy Listing Directories for WordPress ===
+Contributors: businessdirectoryplugin
```

Comment:
Would you mind changing this to:
Contributors: businessdirectoryplugin, sswells, srwells 

Then we can close my other open PR.

Reply:
Of course!